apps/calendar/test/unit/support/calendar.js

```diff
@@ -0,0 +1,143 @@
+define(function(require, exports) {
```

Comment:
Moved all of the stuff from `setup.js` that used to be `window.testSupport.calendar` to its own module